cl/favorites/views.py

```diff
@@ -51,8 +53,10 @@ def get_note(request: HttpRequest) -> HttpResponse:
     return note
 
 
+@sync_to_async
 @login_required
-def save_or_update_note(request: HttpRequest) -> HttpResponse:
+@async_to_sync
+async def save_or_update_note(request: HttpRequest) -> HttpResponse:
```

Comment:
Kinda gross. I guess the problem is that login_required doesn't support async yet? 

I suppose this doesn't accomplish the same thing? 

```
@sync_to_async
@login_required
def save_or_update_note(request: HttpRequest) -> HttpResponse:
```

Reply:
> Kinda gross. I guess the problem is that login_required doesn't support async yet?

Yeah, maybe should make an issue on the django issue tracker to see if they can make `login_required` handle async natively.

> I suppose this doesn't accomplish the same thing?

Not really since the function isn't async then.